lib/galaxy/structured_app.py

```diff
@@ -83,7 +84,7 @@ class MinimalManagerApp(MinimalApp):
     role_manager: Any  # 'galaxy.managers.roles.RoleManager'
     installed_repository_manager: Any  # 'galaxy.tool_shed.galaxy_install.installed_repository_manager.InstalledRepositoryManager'
     user_manager: Any
-    job_config: Any  # 'galaxy.jobs.JobConfiguration'
+    job_config: "JobConfiguration"
```

Comment:
Out of curiosity what's the distinction here for when the types are quoted?

Reply:
It's a trick to workaround circular imports needed only for type checking.